### DISCUSSION TITLE
Is the scene fixed until the end of competition?

### DISCUSSION THREAD

**Discussion by MJavadZallaghi** [cyberbotics/wrestling#36] [Q&A] [answered]
Hi @omichel. I'm working on AI-based object detection capability for my robot.
I have trained a model based on collected data (images) in `Orthographic Projection` view settings for a higher speed of simulation, which causes a dark background in the simulation scene and consequently the collected data. I noticed that when the view is set to `Perspective Projection`, which is happening in the server simulations settings, my trained model is not performing well in the detection of objects in the scene.
So, I have to do the model training for a new collected data in `Perspective Projection` view settings. It's important for me to know that either if the image and background scene will remain in the current settings (a room with the current near-blue color on the walls, a door and artwork on the wall, etc.) until the end of the competition or not. This is because of the highly time-consuming process of training AI models with new data.
Thanks.

**Comment by omichel** [comment, marked as answer]
Yes, I guarantee that we are not going to change the background of the simulation until the end of the competition as we are aware that it could potentially break the vision algorithms of many participants.
More generally, we are trying to make as few changes as possible and only minor changes in the rules during the competition to avoid breaking the performance of the participants.
The more we advance in the competition, the more unlikely we will make change to the rules.
